src/auth/auth.service.ts

```diff
@@ -372,4 +374,70 @@ export class AuthService {
       return internalServerError();
     }
   }
+
+  async deleteUserByUserId(
+    userId: number,
+    id: number,
+  ): Promise<void | CustomException> {
+    if (userId !== id) {
+      return forbidden();
+    }
+
+    const user = await this.prisma.user.findUnique({
+      where: {
+        id,
+        isDeleted: false,
+      },
+    });
```

Comment:
```suggestion
    const user = await this.userService.getUserById(id);
```

`users.service.ts` 에 해당 역할을 하는 함수가 똑같이 구현되어 있어요 user에 대한 질의를 하는 부분은 users 의 서비스 레이어가 담당하는 게 좋을 것 같습니다.

Reply:
반영하겠습니다 ~